src/index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Leaderboard</title>
+</head>
+<body>
+<div class="main">
+    <h1>Leaderboard</h1>
+    <div class="content">
+    <div class="score">
+        <div class="scoreHeader">
+        <h2>Recent Scores</h2>
+        <button>Refresh</button>
+    </div>
+        <p class="recentScore" id="recentScores">
+            Name: 100 <br>
+            Name: 20 <br>
+            Name: 50 <br>
+            Name: 125 <br> 
+        </p>
```

Comment:
- [ ] good job making your project look like the wireframe 👍 however according to the requirements, you should use HTML just to make the layout. To view the items inside the list you are expected to use Javascript/ES6 modules, with import and export. as mentioned in [the assignment](https://github.com/microverseinc/curriculum-javascript/blob/main/leaderboard/m1_basic_structure.md)

Reply:
This issue has been resolved by generating the list using JavaScript